OpenRA.Game/Server/Server.cs

```diff
@@ -392,6 +392,10 @@ public Server(List<IPEndPoint> endpoints, ServerSettings settings, ModData modDa
 				foreach (var t in serverTraits.WithInterface<INotifyServerShutdown>())
 					t.ServerShutdown(this);
 
+				// Force a cleanup to close all connections
```

Comment:
```suggestion
				// Cleanup all connections
```

Reply:
The wording here was rather intentional: Just clearing the list does not enforce any sort of cleanup, we have to call `Dispose` ourselves to force-close the connections. Maybe this can be worded differently. If you insist on this change, I would just remove the comment as Imo `// Cleanup all connections` does tell me the same as no comment.

Reply:
For me as someone who doesn't know how the rest of the code works, this comment is nothing but confusing

Reply:
Would `// Dispose must be called on a connection to close the underlying socket` be better?

Reply:
perhaps
`// Make sure to close connections after the server is shutdown, we don't want to keep clients waiting`

Reply:
Sounds reasonable, thanks for the suggestion.